db/migrate_cards/20161005120800_answer_lookup_table.rb

```diff
@@ -15,7 +15,7 @@ def up
       t.string :title_name
       t.string :record_name
       t.string :value
-      t.float :numeric_value
+      t.decimal :numeric_value, precision: 30, scale: 5
```

Comment:
with float AR returned values that didn't match db content
eg numeric value is 234669 and `answer.numeric_value` returns 234660.0